### ISSUE TITLE
Link to haxeui-templates leading to 404

### ISSUE THREAD

**Issue by azrafe7** [haxeui/haxeui-html5#36]
Screenshot:

![image](https://user-images.githubusercontent.com/1266447/124403802-62c64480-dd38-11eb-9fb6-04336a3ed662.png)

This probably also affects other haxeui repos.

Cheers, azrafe7

**Comment by ianharrigan** [comment]
Good point... thats all been replaced by the CLI tool now. Ill make a note to clean it all up and make it more correct.

Cheers,
Ian

FYI: `haxelib run haxeui-core help` is a starting point
